test/ANNOTATIONS.yaml

```diff
@@ -670,6 +678,8 @@ mg-b:
     - Move chpl_getPrivatizedClass() into chpl-privatization.h (#6212)
   05/23/17:
     - Fix NPB MG (#6284)
+  06/13/17:
+    - Optimize updateFluff (#6418)
```

Comment:
@psahabu not on you, but think this graph is misnamed. `mg-b` is the single-node version, and all of these annotations are 16-node so `mg-b` should probably be renamed to `mg.ml-time`